charts/fluentd/templates/daemonset.yaml

```diff
@@ -148,6 +149,7 @@ spec:
         k8s-app: {{ .Values.k8sApp }}
         version: v1
     spec:
+      priorityClassName: {{ .Values.windowsDaemonset.priorityClassName }}
```

Comment:
Please add a condition that adds this field if the value is set.

Reply:
Fixed